### PR TITLE
bugs

### PR DESCRIPTION
**Pull request by LukeSettle**
Fixed bug where pressing enter sends the form when trying to search location.
Moved modal openers to nav for profile page.

### CODE REVIEW

File: app/views/refills/_nav.html.erb [hunk 1]
```diff
@@ -32,6 +31,18 @@
           <%end%>
         </form>
       </div>
+      <%elsif current_page?(controller: 'events', action: 'new')%>
```
Comment (kurtisnelson) [inline app/views/refills/_nav.html.erb]: does this line need to exist?

Comment (LukeSettle) [inline app/views/refills/_nav.html.erb]: Yes. current_page? was being really weird

Comment (kurtisnelson) [inline app/views/refills/_nav.html.erb]: can you maybe move this component outside of the shared nav partial and only render it on pages you need it?

Comment (LukeSettle) [inline app/views/refills/_nav.html.erb]: Ehh

**Pull request merged**